ajax/libs/Tipue-Search/package.json

```diff
@@ -7,27 +7,11 @@
     "search",
     "site-search"
   ],
-  "version": "5.0.0",
+  "version": "7.1.0",
   "author": {
     "name": "Tipue",
     "url": "http://www.tipue.com"
   },
   "license": "MIT",
-  "homepage": "http://www.tipue.com/search/",
-  "autoupdate": {
```

Comment:
Out of curiosity, why did you decide to disable auto-updating?

Reply:
The repo no longer exists it seems, I would assume that's why :)

Reply:
Yep. I couldn't find any links on the [website](http://www.tipue.com/search/) pointing to a repository. Only a download link to this version is available. The package is still under MIT license.